src/qt/coincontroldialog.cpp

```diff
@@ -129,11 +126,9 @@ CoinControlDialog::CoinControlDialog(const PlatformStyle *platformStyle, QWidget
     ui->treeWidget->setColumnWidth(COLUMN_ADDRESS, 290);
     ui->treeWidget->setColumnWidth(COLUMN_DATE, 110);
     ui->treeWidget->setColumnWidth(COLUMN_CONFIRMATIONS, 100);
-    ui->treeWidget->setColumnWidth(COLUMN_PRIORITY, 100);
```

Comment:
Thanks for the help!

With resetguisettings I am getting this now:

![screenshot from 2016-08-24 11-26-12](https://cloud.githubusercontent.com/assets/6399679/17925541/a54f4cde-69ed-11e6-8670-0de75e089bf6.png)